odoo/addons/base/models/res_partner.py

```diff
@@ -147,6 +148,10 @@ def _name_search(self, name, domain=None, operator='ilike', limit=None, order=No
             domain = [('name', operator, name)] + domain
         return self._search(domain, limit=limit, order=order)
 
+    @api.model
+    def _search_display_name(self, operator, value, limit=None):
```

Comment:
Shoudln't it be done in ``_name_search`` ? as its docstring says
```
    Private implementation of name_search, returning ids or a :class:`Query` object.
```
And name_search docstring says
```
    Search for records that have a display name matching the given
    ``name`` pattern when compared with the given ``operator``, while also
    matching the optional search domain (``args``).
```